src/pages/docs/platform/architecture/edge-network.mdx

```diff
@@ -0,0 +1,144 @@
+
+These client-side mechanisms work in conjunction with the server-side infrastructure to provide reliable connectivity. By providing automated reconnection, and automated use of fallback endpoints at the client level, Ably can maintain service availability even during significant infrastructure disruptions that might otherwise cause extended outages.
+
+### Infrastructure redundancy
+
+Beyond DNS and client-side fallbacks, Ably's infrastructure includes multiple layers of redundancy to ensure continuous service availability. Each datacenter contains redundant servers, network paths, and storage systems to eliminate single points of failure within the datacenter.
+
+The multi-region architecture provides an additional layer of redundancy, as the failure of an entire datacenter does not impact the availability of the service as a whole. Clients can continue to connect via other datacenters, albeit potentially with increased latency.
+
+The use of CloudFront as the front-end distribution layer adds another level of redundancy, as CloudFront itself is designed to be highly available, with redundant capacity across multiple edge locations. This means that even if specific edge locations experience issues, clients can still connect via other edge points.
+
+## Protection against denial of service attacks
+
+As a critical infrastructure provider for many businesses, Ably is a potential target for denial of service (DoS) attacks. The edge network includes multiple layers of protection to detect, mitigate, and absorb such attacks without impacting legitimate traffic.
+
+### Layer 3/4 DoS protection with AWS Shield Advanced
+
+Ably utilizes AWS Shield Advanced to protect against volumetric DDoS attacks at the network and transport layers (Layers 3 and 4). These attacks typically involve flooding the target with a high volume of packets or connection attempts, aiming to exhaust network bandwidth or connection handling capacity.
+
+AWS Shield Advanced provides automatic attack detection by continuously monitoring traffic patterns to detect anomalies that indicate an attack is underway. When an attack is detected, Shield automatically applies mitigations to filter malicious traffic while allowing legitimate traffic to pass through.
+
+During large-scale attacks, AWS can leverage its global network infrastructure to absorb and diffuse attack traffic, preventing it from reaching Ably's resources. AWS Shield includes access to the AWS DDoS Response Team (DRT), providing expert assistance during attacks.
+
+These protections ensure that even large-scale volumetric attacks, which might otherwise overwhelm traditional network defenses, can be effectively mitigated before reaching Ably's infrastructure.
+
+### Layer 7 DoS protection with CloudFront WAF
```

Comment:
Layer 7/WAF is not just about DoS, it's there to block malicious and spurious (bot) traffic so we could reframe this section with that in mind.